src/models/DataSetStore.js

```diff
@@ -126,12 +141,61 @@ export default class DataSetStore {
         this.updateFromAssociations(fieldPath, oldValue);
     }
 
+    saveDataset() {
+        return new Promise(async (complete, error) => {
+            // maintenance-app uses a custom function to save the dataset as it needs to do
+            // some processing  not allowed by dataset.save(). The code in this method is copied
+            // from maintenance-app/src/EditModel/objectActions.js
```

Comment:
@tokland and what is the difference from what we have before? There were some properties that were not being saved before?